### PR TITLE
batch addresses in box resolver

### LINKED ISSUE #43: Batch get boxes by addresses

### PR DESCRIPTION
**Pull request by SepehrGanji**

### CODE REVIEW

File: src/graphql/resolvers/box-resolver.ts [hunk 1]
```diff
@@ -69,16 +69,26 @@ class BoxesQueryArgs {
   @Field(() => String, { nullable: true })
   tokenId?: string;
 
+  /** @deprecated */
   @ValidateIf((o: BoxesQueryArgs) => isDefined(o.ergoTree))
   @IsEmpty({ message: REDUNDANT_QUERY_MESSAGE })
```
Comment (capt-nemo429) [inline src/graphql/resolvers/box-resolver.ts]: Your implementation on `BoxRepository` allows the usage of `address` and `ergoTree` without conflicts, so I think we can remove this validation.

File: src/graphql/resolvers/box-resolver.ts [hunk 2]
```diff
@@ -69,16 +69,26 @@ class BoxesQueryArgs {
...
   address?: string;
 
+  @Field(() => [String], { nullable: true })
+  @ArrayMaxSize(20)
+  addresses?: string[];
+
+  /** @deprecated */
   @ValidateIf((o: BoxesQueryArgs) => isDefined(o.address))
   @IsEmpty({ message: REDUNDANT_QUERY_MESSAGE })
   @Field(() => String, { nullable: true })
```
Comment (capt-nemo429) [inline src/graphql/resolvers/box-resolver.ts:86]: Same as `address`

**Review by capt-nemo429 — CHANGES_REQUESTED** [review]
Awesome work on `BoxRepository`!

File: src/graphql/resolvers/box-resolver.ts (hunk 1 above)
Comment (SepehrGanji) [reply src/graphql/resolvers/box-resolver.ts]: Removed.

File: src/graphql/resolvers/box-resolver.ts (hunk 2 above)
Comment (SepehrGanji) [reply src/graphql/resolvers/box-resolver.ts:86]: Removed.

**Review by capt-nemo429 — APPROVED** [review]

**Pull request merged**